packages/backend/src/models/SpaceModel.ts

```diff
@@ -253,6 +331,7 @@ export class SpaceModel {
         const [space] = await this.database(SpaceTableName)
             .insert({
                 project_id: project.project_id,
+                is_private: true,
```

Comment:
We should have `is_private: false; // TODO change to true once we support private spaces in the UI` and have the correct permission logic in the SpaceService `hasSpaceAccess`. 
Otherwise all the spaces they create during this time with suddenly appear as private once we support it in the FE.